samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>Important metrics to look at while measuring the performance of the&nbsp;<code>EntityRecognizer</code>&nbsp;model are&nbsp;<strong>Precision</strong>,&nbsp;<strong>Recall</strong>,&nbsp;and&nbsp;<strong>F1-measures</strong>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>